tutorials/test_status/runtime_status_test.yml

```diff
@@ -49,3 +49,24 @@ buildspecs:
     status:
       runtime:
         max: 1.0
+
+  timelimit_max_specified:
+    type: script
+    executor: generic.local.sh
+    description: "Run a sleep job for 2 seconds and test pass if it's within the specified maximum time"
+    tags: ["tutorials"]
+    run: sleep 2
+    status:
+      runtime:
+        max: 4.0
+
+  timelimit_min_max_specified:
+    type: script
+    executor: generic.local.sh
+    description: "Run a sleep job for 2 seconds and test pass if it's within the specified minimum and maximum time"
+    tags: ["tutorials"]
+    run: sleep 2
+    status:
+      runtime:
+        min: 1.5
+        max: 3.5
```

Comment:
i dont understand why we have these two tests. The above 5 test cover this already and can be used as part of the regression test. Please remove these tests unless i am missing something. 

Also if test needs to be added to buildspec for regression test we should not add them to the file where we created symbolic link because this will affect the user documentation. Instead create a separate test in the same directory where test file is.

Reply:
Noted, thanks!

Reply:
@shahzebsiddiqui fixed.